bundle/Form/DataMapper.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form;
+
+use DateTime;
+use Symfony\Component\Form\DataMapperInterface;
+use Symfony\Component\Form\Exception\UnexpectedTypeException;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use Symfony\Component\PropertyAccess\PropertyPathInterface;
+use function is_array;
+use function is_object;
+
+/**
+ * A data mapper using property paths to read/write data.
+ */
+abstract class DataMapper implements DataMapperInterface
+{
+    protected FieldTypeHandlerRegistry $fieldTypeHandlerRegistry;
+
+    protected PropertyAccessorInterface $propertyAccessor;
+
+    /**
+     * Creates a new property path mapper.
+     */
+    public function __construct(
+        FieldTypeHandlerRegistry $fieldTypeHandlerRegistry,
+        ?PropertyAccessorInterface $propertyAccessor = null
+    ) {
+        $this->fieldTypeHandlerRegistry = $fieldTypeHandlerRegistry;
+        $this->propertyAccessor = $propertyAccessor ?: PropertyAccess::createPropertyAccessor();
+    }
+
+    public function mapDataToForms($data, $forms): void
```

Comment:
Use signature from the interface here.

Reply:
Solved in d42fe134

Reply:
Do not fix multiple thinks in the same commit, also parameter name is changed from the interface.

Reply:
solved in 83093de5